File: Level1/Get Average.cpp
/*
    https://programmers.co.kr/learn/courses/30/lessons/12944
*/
#include <vector>
#include <numeric>

using namespace std;

double solution(vector<int> arr) {
    double total = accumulate(arr.begin(), arr.end(), 0);
    
    return total / arr.size();
}

File: Java/Level0/get average.java
/*
	https://school.programmers.co.kr/learn/courses/30/lessons/12944?language=java
*/

class Solution {
    public double solution(int[] arr) {
        double answer = 0;
        
        for(int n : arr)
        {
            answer += (double)n;
        }
        
        return answer / (double)arr.length;
    }
}